velox/functions/prestosql/registration/StringFunctionsRegistration.cpp

```diff
@@ -42,6 +42,8 @@ void registerSimpleFunctions(const std::string& prefix) {
   registerFunction<LevenshteinDistanceFunction, int64_t, Varchar, Varchar>(
       {prefix + "levenshtein_distance"});
   registerFunction<LengthFunction, int64_t, Varchar>({prefix + "length"});
+  registerFunction<XxHash64StringFunction, int64_t, Varchar>(
+      {prefix + "xxhash64"});
```

Comment:
Let's register it under internal names like `$operator$xx_hash_64` so that user won't invoke them directly